packages/bpk-docs/src/layouts/links.js

```diff
@@ -57,6 +57,11 @@ const componentsLinks = [
       { id: 'BADGE', route: routes.BADGE, children: 'Badge' },
       { id: 'PANELS', route: routes.PANELS, children: 'Panels' },
       { id: 'IMAGES', route: routes.IMAGES, children: 'Images' },
+      {
+        id: 'INFINITE_SCROLL',
+        route: routes.INFINITE_SCROLL,
+        children: 'Infinite Scroll',
```

Comment:
Can we change to sentence case i.e. 'Infinite scroll'?

Reply:
done